imports/ui_3/views/components/topic-vote-box.html

```diff
@@ -67,22 +67,44 @@ <h2 class="m-t-none m-b-xs">{{title}}</h2>
           {{else}}
           <!-- voting results -->
             <h4 class="text-center">
+            <br>
             {{#if equals this.vote.type "preferential"}}
               {{_ 'Support of options'}}:            
             {{else}}
               {{_ 'Aggregation of votes'}}:
             {{/if}}
             </h4>
-            <div class="row">
-              <div class="col-lg-12 text-center">
-                {{#each sum in this.voteSummaryDisplay}}
-                <div>{{_ sum.choice}}: {{sum.votingUnits}} ({{displayPercent sum.percentOfVotes}})</div>
-                {{/each}}
+            {{#unless equals votedCount 0}}
+              <div class="row">
+                <div class="col-lg-2 col-md-1"></div>
+                <div class="col-lg-8 col-md-10" style="padding-bottom: 20px">
                 {{#if Template.subscriptionsReady}}
                   {{> Chart type=chartType data=chartData options=chartOptions height="200"}}
                 {{/if}}
+                </div>
+              </div> 
+              <div class="row">
+                <div class="col-lg-2 col-md-1"></div>
+                <div class="col-lg-8 col-md-10">
+                  <div class="table-responsive">
+                    <table class="table table-striped table-hover">
+                      <tbody> 
+                        {{#each sum in this.voteSummaryDisplay}}
+                          <tr>
+                            <td>{{_ sum.choice}}:</td>
+                            <td class="nowrap text-center">{{sum.votingUnits}}</td>
```

Comment:
Itt kene lekerekiteni, mert az egy megjelenitesi kerdes, hogy kerekitve irom ki, nem egy szamitasi kerdes.

Reply:
Azt hiszem van is ra helper az utils-ban